delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/marshalling/RdfMarshallingSpec.scala

```diff
@@ -9,7 +9,7 @@ import ch.epfl.bluebrain.nexus.delta.SimpleResource.{context, contextIri}
 import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.nxv
 import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolution
 import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
-import ch.epfl.bluebrain.nexus.delta.routes.marshalling.RdfMediaTypes._
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfMediaTypes._
```

Comment:
Maybe this one too

Reply:
that's for jsonLd stuff. For now this is still defined in the app.